tasks/create_storage_domain.yml

```diff
@@ -1,10 +1,18 @@
 ---
 - name: Create hosted engine local vm
   block:
-  - name: Wait until all interfaces wake up
-    wait_for:
-      timeout: 150
-      delegate_to: "{{ he_ansible_host_name }}"
+  - name: Wake up the storage interface
+    command: ip link set '{{ he_storage_if }}' up
```

Comment:
we have to assume this interface is pre-configured by the user, in such case, it should be activated automatically,
I don't see the point to activate it again.